### PR TITLE
update the spec to match the new proxy endpoint behaviour

### PR DESCRIPTION
**Pull request by elliotCamblor**

### CODE REVIEW

File: docs/spec.yaml [hunk 1]
```diff
@@ -282,6 +233,10 @@ components:
               command:
                 type: string
                 description: 'The name of the SDK method to invoke'
+              user:
+                $ref: '#/components/schemas/User'
+              Event:
```
Comment (ajwootto) [inline docs/spec.yaml:238]: should this be lowercase

File: proxies/nodejs/handlers/user.ts [hunk 2]
```diff
@@ -1,26 +0,0 @@
-import Koa from 'koa'
```
Comment (ajwootto) [inline proxies/nodejs/handlers/user.ts:1]: if you're deleting this theres also a c# equivalent still isn't there?

Comment (elliotCamblor) [reply proxies/nodejs/handlers/user.ts:1]: theres not

Comment (elliotCamblor) [reply proxies/nodejs/handlers/user.ts:1]: but there is a HelloWorld controller that I deleted

**Review by ajwootto — APPROVED** [review]

**Pull request merged**